packages/constants/src/lib/constants/constants.ts

```diff
@@ -804,8 +804,9 @@ export const HTTP_BY_NETWORK: Record<
   habanero: HTTPS,
   'datil-dev': HTTPS,
   'datil-test': HTTPS,
-  custom: HTTP,
-  localhost: HTTP,
+  internalDev: HTTPS,
+  custom: HTTP, // default, can be changed by config
+  localhost: HTTP, // default, can be changed by config
```

Comment:
Nice, do we think wrapping the `PROTO` type in an object for future config values is worth it at this time?

Reply:
the only type right now is `443` right? not sure if we have `80` though as they all range from 7xxx to 8xxx

Reply:
We might want to add a port override for local testing. This could become handy with future tooling.

Reply:
Actually think just having the single property is okay for now. But I think we will need to think about overrides and how we handle them broadly speaking.